.github/workflows/release-mac.yml

```
@@ -210,17 +210,48 @@ jobs:
             ${{ env.DMG_PATH }}
+            ${{ env.APPCAST_PATH }}
           generate_release_notes: true
           draft: false
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: Update Landing Page Appcast
+        if: startsWith(github.ref, 'refs/tags/')
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
```

actionlint (1.7.10)

shellcheck reported issue in this script: SC2086:info:2:4: Double quote to prevent globbing and word splitting

(shellcheck)